packages/core/src/tracing/instrumentation/protocols/httpClient.js

```diff
@@ -101,6 +110,15 @@ function shouldBeBypassed(parentSpan, options) {
   if (parentSpan && parentSpan.n === 'sqs' && isAWSNodeJSHeader && hostMatchesSQS) {
     return true;
   }
+
+  const isGCPNodeJSHeader = evaluateHeaderValue(
+    userAgent,
+    header => header.toLowerCase().indexOf('google-api-nodejs-client') > -1
+  );
+
+  const hostMatchesGPC = options && options.host && options.host.indexOf('googleapis') !== -1;
+  if (isGCPNodeJSHeader && hostMatchesGPC) return true;
+
```

Comment:
_issue:_ I think this is too broad. It will suppress all HTTP exits for GCP, not only the specific requests auth requests for GCP storage that we want to get rid of. Do you have the full HTTP request details and/or the full host name at hand? Maybe we can refine/tweak the conditions here to make the match more precise.